examples/external-data/jest.config.cjs

```diff
@@ -34,4 +34,5 @@ module.exports = {
 			},
 		],
 	],
+	testTimeout: 10000,
```

Comment:
Intentional?

Reply:
If so, we should note it in the PR description

Reply:
Yep, when I removed node-fetch, the tests started failing with timeout errors.

Reply:
That seems like a problem...

Reply:
I think the changes in service.ts introduced an unawaited promise that is likely the culprit. I'll leave a comment there.

Reply:
Yeah, it sounds like we've introduced or exacerbated some sort of race condition.